### ISSUE TITLE
A11y_Visual Studio Code Client_Sep2021_Debug Adapter Protocol_Debug Adapter Protocol specification_Zoom: On 200% Zoom links present in right navigation section will get invisible.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/debug-adapter-protocol#210]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag:**#A11yTCS;#DesktopWeb;#Win10;#WCAG1.4.4;#FTP;#A11yMAS;#ChromiumEdge;#Zoom;#A11ySev1;#P2_Visual Studio Code Clien_Sep2021;#Visual Studio Code Client;#P2_WCP;#AILimited;

## **Environment Details:**

Application Name: Visual Studio Code Client
#URL: https://microsoft.github.io/debug-adapter-protocol
Browser Versions: Microsoft Edge Version 93.0.961.38 (Official build) (64-bit)
Windows Version: Windows10

**Example Page UR**L: https://microsoft.github.io/debug-adapter-protocol/specification

## **Repro Steps:**

1. Open the URL: https://microsoft.github.io/debug-adapter-protocol/specification
2. Debug Adapter Protocol specification screen will get open
3. Zoom screen up to 200% and verify the issue.

## **Actual:**

On 200% Zoom links present in right navigation section will get invisible.

**Same issue is also observed with Scenario "Language Server Protocol Specification" .**

## **Expected:**

All the controls should be visible properly on 200% zoom.

## **User Impact:**

Users will not be able to see all the controls present on screen.

## **Attachment:**
![#210](https://user-images.githubusercontent.com/60422426/133751728-49aeb59d-f2ca-4180-9f58-d8d1df0aa873.gif)

**Comment by rzhao271** [comment]
Fixed with https://github.com/microsoft/debug-adapter-protocol/commit/ecfcc143c4ea0bfa3048e84599f5b97a89a6a97a.
Also fixed in the LSP site with the recent commits on Nov 8 at https://github.com/microsoft/language-server-protocol/commits/gh-pages.

The commits are unverified because I used github.dev and didn't have code signing set up on it.

**Comment by NiharikaTipirishetty** [comment]
#P2_debugadapterprotocol_Web_Sep2021;

**Comment by prabhudevu** [comment]
I have verified the issue in the Env: https://microsoft.github.io/debug-adapter-protocol/specification,
Issue is fixed, added attachment for reference.

![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/98821792/178431306-48464c74-3565-457b-8a9c-ffff453a0d4f.png)

**Comment by prabhudevu** [comment]
#closed;
